src/data/markdown/translated-guides/en/03 Results output/200 Real-time/00 Prometheus remote write.md

```diff
@@ -31,108 +32,153 @@ All k6 metric types are converted into an equivalent Prometheus remote write typ
 | Counter | Counter                   | `k6_*_total`         |
 | Gauge   | Gauge                     | `k6_*_<unit-suffix>` |
 | Rate    | Gauge                     | `k6_*_rate`          |
-| Trend   | Gauges / Native Histogram | `k6_*_<unit-suffix>` |
+| Trend   | [Gauges (default)](#trend-stats-option) or [Native Histogram](#prometheus-native-histogram) | `k6_*_<unit-suffix>` |
+
+The output maps the metrics into time series with Name labels.
+As much as possible, k6 respects the [naming best practices](https://prometheus.io/docs/practices/naming) that the Prometheus project defines:
+
+* All time series are prefixed with the `k6_` namespace.
+* All time series are suffixed with the base unit of the sample value (if k6 knows what the base unit is).
+* Trends and rates have the relative suffixes, to make them more discoverable.
+
+## Trend metric conversions
+
+This output provides two distinct mechanisms to send [built-in and custom Trend metrics](https://k6.io/docs/using-k6/metrics/) to Prometheus:
+
+1. [Trend stats option](#trend-stats-option) 
+2. [Prometheus Native histogram](#prometheus-native-histogram) 
+
+Both options provide efficient storage of test results while providing high-precision queries.
+
+Note that k6 aggregates trend metric data before sending it to Prometheus in both options. The reasons for aggregating data are:
+
+- Prometheus stores data in a millisecond precision (`ms`), but k6 metrics collect data points with higher accuracy, nanosecond (`ns`).
+- A load test could generate vast amounts of data points. High-precision raw data could quickly become expensive and complex to scale and is unnecessary when analyzing performance trends.
+
+### Trend stats option
+
+By default, Prometheus supports [Counter and Gauge Metric types](https://prometheus.io/docs/concepts/metric_types/). Therefore, the trend stats option is the default of this output and converts all the k6 trend metrics to Counter and Gauges Prometheus metrics. 
+
+The [`K6_PROMETHEUS_RW_TREND_STATS` option](#options) accepts a comma-separated list of stats functions that define how to convert all the k6 trend metrics:
 
-### Trend metric conversion
+- `count`
+- `sum`
+- `min`
+- `max`
+- `avg`
+- `med`
+- `p(x)`
```

Comment:
The rendered version uses a lot of space. I'm for a single-line list.

Reply:
I reordered the paragraphs for this:

https://github.com/grafana/k6-docs/pull/1085/commits/d8fa71fcc3b3540cd6ba2a8532ef8427588187d5